pandas/core/missing.py

```diff
@@ -223,13 +223,22 @@ def _interpolate_scipy_wrapper(x, y, new_x, method, fill_value=None,
         raise ImportError('{0} interpolation requires Scipy'.format(method))
 
     new_x = np.asarray(new_x)
-
-    # ignores some kwargs that could be passed along.
-    alt_methods = {
-        'barycentric': interpolate.barycentric_interpolate,
-        'krogh': interpolate.krogh_interpolate,
-        'piecewise_polynomial': interpolate.piecewise_polynomial_interpolate,
-    }
+    import scipy
+    if scipy.__version__ <= '0.17':
```

Comment:
Also, maybe start with

```python
alt_methods = {
      'barycentric': interpolate.barycentric_interpolate,
      'krogh': interpolate.krogh_interpolate,
}
```

and then do your if block checking the version. That way you aren't repeating the other two methods.